packaging/installer/methods/macos.md

```diff
@@ -13,8 +17,10 @@ configuration](/streaming/README.md).
 We recommend installing Netdata with the community-created and -maintained [**Homebrew
 package**](#install-netdata-with-the-homebrew-package). 
```

Comment:
If we're going to recommend using Docker (or kickstart), then we should remove this sentence.

Reply:
In terms of user experience, the Homebrew method is _far_ superior to the kickstart script or Docker, the only caveat is the lack of cloud support.